css/style.css

```diff
@@ -0,0 +1,244 @@
+
+.nav-bar .icons {
+  border-radius: 4px;
+  height: 40px;
+  width: 900px;
+  margin: 0 auto;
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  z-index: 5;
+  margin-top: 0;
+}
+
+.nav-bar fieldset {
+  float: left;
+  height: 40px;
+}
+
+.icons-list .field-icons {
+  width: 95px;
+  border: none;
+  border-right: 1px solid #a8a8a8;
+}
+
+.field-search-bar .field-search-icons {
+  width: 100px;
+  float: right;
+  margin-top: 10px;
+  align-items: center;
+  border: none;
+  border-right: 1px solid #a8a8a8;
+}
+
+.iphone-img > li {
+  padding: 0;
+  list-style: none;
+}
+
+.footer-copyright li {
+  list-style-type: none;
+  float: left;
+}
+
+.icons ul li {
+  list-style-type: none;
+}
+
+footer a {
+  color: rgb(0, 136, 204);
+  cursor: pointer;
+}
+
+.content a {
+  text-decoration: none;
+}
+
+.footer-copyright li > a {
+  text-decoration: none;
+}
+
+.icons ul li a {
+  text-decoration: none;
+  margin: auto;
+  position: relative;
+  top: 9px;
+}
+
+.field-search-icons .search-wrapper {
+  position: absolute;
+}
+
+.field-search-icons .search-wrapper .fa {
+  position: absolute;
+  top: 5px;
+  left: 6px;
+  bottom: 6px;
+}
+
+.field-search-bar .search-wrapper .search-input {
+  border-radius: 10px;
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFW…px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  width: 80px;
+}
+
+.content {
+  display: flex;
+  text-align: center;
+  font-weight: 100;
+}
+
+.header-primary {
+  font-size: 3rem;
+  font-weight: 100;
+}
+
+.header-secondary {
+  font-size: 1.4rem;
+  line-height: 1.6;
+  padding-top: 20px;
+}
+
+.content-container {
+  min-width: 500px;
+  margin: auto;
+  position: relative;
+  top: 5px;
+}
+
+.footer ul li > a {
+  text-decoration: none;
+}
+
+.footer .footer-links a {
+  text-decoration: none;
+}
```

Comment:
Merge into one style. You can also combine these with the style in lines 110 - 116 since its the same single style for all of them.